### PR TITLE
add to `make_labelled()` vignette

### LINKED ISSUE #82: [FEATURE] Add `make_labelled()` usage to vignette
# Feature Request Description
Update the main `REDCapTidieR` vignette with a section showing off usage of `make_labelled()` and exported formatters

### Checklist

- [x] The issue is atomic
- [x] The issue description is documented
- [x] The issue title describes the problem succinctly
- [x] Developers are assigned to the issue
- [x] Labels are assigned to the issue

### PR DESCRIPTION
**Pull request by ezraporter**
# Description
This PR updates our main `REDCapTidieR` vignettes with a section describing how to use `make_labelled()`

# Proposed Changes 

- Add section to `REDCapTidieR.Rmd`
- Minor updates to `make_labelled()` roxygen docs and pkgdown

### Issue Addressed
Closes #82

### PR Checklist
Before submitting this PR, please check and verify below that the submission meets the below criteria:

- [x] New/revised functions have associated tests
- [x] New/revised functions that update downstream outputs have associated static testing files (`.RDS`) updated under `inst/testdata/create_test_data.R`
- [x] New tests that make API calls use `httptest::with_mock_api` and any new mocks were added to `tests/testthat/fixtures/create_httptest_mocks.R`
- [x] New/revised functions use appropriate naming conventions
- [x] New/revised functions don't repeat code
- [x] Code changes are less than **250** lines total
- [x] Issues linked to the PR using [GitHub's list of keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] The appropriate reviewer is assigned to the PR
- [x] The appropriate developers are assigned to the PR

# Code Review
This section to be used by the reviewer and developers during Code Review after PR submission

### Code Review Checklist

- [ ] I checked that new files follow naming conventions and are in the right place
- [ ] I checked that documentation is complete, clear, and without typos
- [ ] I added/edited comments to explain "why" not "how"
- [ ] I checked that all new variable and function names follow naming conventions
- [ ] I checked that new tests have been written for key business logic and/or bugs that this PR fixes
- [ ] I checked that new tests address important edge cases

### CODE REVIEW

File: vignettes/REDCapTidieR.Rmd [hunk 1]
```diff
@@ -186,6 +187,60 @@ In summary, here are the rules by which `REDCapTidieR` constructs tibbles:
...
+
+This operation is common enough that `REDCapTidier` comes with a built-in helper function.
+
+```{r}
+fmt_strip_trailing_colon("Super Power:")
+```
+
+See the other available helpers with `` ?`format-helpers` ``.
+
+`format_labels` will accept any function that takes a vector of raw labels and returns a vector of formatted labels of the same length. You can also pass it a list functions to be executed on the raw labels in order.
```
Comment (rsh52) [inline vignettes/REDCapTidieR.Rmd:234]: ~"You can also pass it a list functions to be executed on the raw labels in order."~
There's an "of" missing in there, but I also think the order aspect is important to highlight. Suggestion:

"`format_labels` will also accept a list of functions, but it's important to note that the functions will be executed **in the order they are supplied**."

**Review by rsh52 — APPROVED** [review]
One slight change, otherwise I think this all sounds good and is framed right in the context of being a `REDCapTidieR` description of using labels, and not an introduction to the `labelled` package.

Only other thought, it might even do well as its own separate vignette instead of being lumped into the default "Get started" page.

**Review by skadauke — APPROVED** [review]
This looks really good to me!
